step1-03/demo/index.html

```diff
@@ -1,15 +1,9 @@
 <html>
-  <body>
-    <div
-      class="codepen"
-      data-theme-id="36294"
-      data-prefill
-      data-editable="true"
-      data-height="100%"
-      data-theme-id="1"
-      data-default-tab="js,result"
-    >
-      <pre data-lang="css">
+
+<body>
+  <div class="codepen" data-theme-id="36294" data-prefill data-editable="true" data-height="100%" data-theme-id="1"
```

Comment:
not sure if this should be modified to be on one line instead of multiple lines like in the before diff...
@ecraig12345 ?

Reply:
multiple is cleaner, easier to see each of the config values. in general, this is user facing code, so it doesn't 100% matter.

Reply:
Reformatted into multiple lines. I'm sorry for the mix-up.